mongo/src/main/java/com/redhat/lightblue/mongo/crud/IterateAndUpdate.java

```diff
@@ -48,28 +59,39 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(IterateAndUpdate.class);
 
+    private final int batchSize = 64;
```

Comment:
How about we get the batchsize in constructor now, declare a constant in
mongocrudcontroller and pass that? Later we can get that value from the
configuration.

On Wed, May 25, 2016 at 7:48 AM, Dennis Crissman <notifications@github.com>
wrote:

> In
> mongo/src/main/java/com/redhat/lightblue/mongo/crud/IterateAndUpdate.java
> <https://github.com/lightblue-platform/lightblue-mongo/pull/275#discussion_r64574895>
> :
>
> > @@ -48,28 +59,39 @@
> >
> >      private static final Logger LOGGER = LoggerFactory.getLogger(IterateAndUpdate.class);
> >
> > +    private final int batchSize = 64;
>
> MongoMetadataConstants maybe? At least then we wouldn't loose them later
> when we do go to make it configurable.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-mongo/pull/275/files/ed3af7895cc087df19ff1c158c8fa74144ceb734#r64574895>
>